custom-vu/src/main/kotlin/jces1209/vu/action/WorkOnSearch.kt

```diff
@@ -0,0 +1,121 @@
+package jces1209.vu.action
+
+import com.atlassian.performance.tools.jiraactions.api.SEARCH_WITH_JQL
+import com.atlassian.performance.tools.jiraactions.api.SeededRandom
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.action.Action
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.memories.IssueKeyMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.JqlMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.Memory
+import com.atlassian.performance.tools.jiraactions.api.observation.SearchJqlObservation
+import com.atlassian.performance.tools.jiraactions.api.page.IssueNavigatorPage
+import jces1209.vu.MeasureType
+import jces1209.vu.MeasureType.Companion.SWITCH_BETWEEN_ISSUES_IN_SEARCH_RESULTS
+import jces1209.vu.page.IssueNavigator
+import jces1209.vu.page.customizecolumns.ColumnsEditor
+import java.net.URI
+import javax.json.JsonObject
+
+class WorkOnSearch(
+    private val issueNavigator: IssueNavigator,
+    private val jira: WebJira,
+    private val meter: ActionMeter,
+    private val columnsEditor: ColumnsEditor,
+    private val random: SeededRandom,
+    private val filters: Memory<URI>,
+    private val jqlMemory: JqlMemory,
+    private val issueKeyMemory: IssueKeyMemory,
+    private val searchFilterProbability: Float,
+    private val searchJclProbability: Float,
+    private val globalSearchProbability: Float,
+    private val customizeColumnsProbability: Float,
+    private val switchBetweenIssuesProbability: Float
+) : Action {
+    override fun run() {
+        if (roll(switchBetweenIssuesProbability)) {
+            switchBetweenIssues()
+        }
+        if (roll(globalSearchProbability)) {
+            openGlobalIssueSearch()
+        }
+        if (roll(customizeColumnsProbability)) {
+            customizeColumns()
+        }
+        if (roll(searchFilterProbability)) {
+            searchFilter()
+        }
+        if (roll(searchJclProbability)) {
+            searchJcl()
+        }
+    }
+
+    private fun roll(
+        probability: Float
+    ): Boolean = (random.random.nextFloat() < probability)
+
+    private fun switchBetweenIssues() {
+        jira.goToIssueNavigator("resolution = Unresolved ORDER BY priority DESC")
```

Comment:
We should avoid reloading the page all the time. We can go to the page at the beginning and the only use the loaded page as is. We do it in `WorkOnIssue`. This is a more realistic user behavior. And it could help quantify perf improvements like [single-page-apps](https://hello.atlassian.net/wiki/spaces/JFE/pages/765865328/GIN+SPA+Performance).